DQM/L1TMonitorClient/python/L1TStage2uGTClient_cff.py

```diff
@@ -4,21 +4,59 @@
 # directory path shortening
 ugtDqmDir = 'L1T/L1TStage2uGT'
 # input histograms
-errHistNumStr = 'errorSummaryNum'
-errHistDenStr = 'errorSummaryDen'
+errHistNumStr = 'first_bunch_in_train'
+errHistDenStr = 'den_first_bunch_in_train'
 
 # Muons
-l1tStage2uGMTOutVsuGTInRatioClient = DQMEDHarvester("L1TStage2RatioClient",
-    monitorDir = cms.untracked.string(ugtDqmDir+'/uGMToutput_vs_uGTinput'),
-    inputNum = cms.untracked.string(ugtDqmDir+'/uGMToutput_vs_uGTinput/'+errHistNumStr),
-    inputDen = cms.untracked.string(ugtDqmDir+'/uGMToutput_vs_uGTinput/'+errHistDenStr),
-    ratioName = cms.untracked.string('mismatchRatio'),
-    ratioTitle = cms.untracked.string('Summary of mismatch rates between uGMT output muons and uGT input muons'),
-    yAxisTitle = cms.untracked.string('# mismatch / # total'),
-    binomialErr = cms.untracked.bool(True)
+#l1tStage2uGMTOutVsuGTInRatioClient = DQMEDHarvester("L1TStage2RatioClient",
+#    monitorDir = cms.untracked.string(ugtDqmDir+'/uGMToutput_vs_uGTinput'),
+#    inputNum = cms.untracked.string(ugtDqmDir+'/uGMToutput_vs_uGTinput/'+errHistNumStr),
+#    inputDen = cms.untracked.string(ugtDqmDir+'/uGMToutput_vs_uGTinput/'+errHistDenStr),
+#    ratioName = cms.untracked.string('mismatchRatio'),
+#    ratioTitle = cms.untracked.string('Summary of mismatch rates between uGMT output muons and uGT input muons'),
+#    yAxisTitle = cms.untracked.string('# mismatch / # total'),
+#    binomialErr = cms.untracked.bool(True)
+#)
+
+#dqmRatioClient = DQMEDHarvester("DQMGenericClient",
+#    subDirs = cms.untracked.vstring("L1T/L1TStage2uGT/"),
+#    efficiency = cms.vstring(
+#       "Ratio_First_Bunch_In_Train 'Trigger Bits vs BX' first_bunch_in_train den_first_bunch_in_train",
+#       "Ratio_Last_Bunch_In_Train 'Trigger Bits vs BX' last_bunch_in_train den_last_bunch_in_train",
+#       "Ratio_Isolated_Bunch_In_Train 'Trigger Bits vs BX' isolated_bunch den_isolated_bunch_in_train",
+#
+#    ),
+#    resolution = cms.vstring(),
+#    outputFileName = cms.untracked.string(""),
+#    verbose = cms.untracked.uint32(0)
+#)
+
+#L1GenPostProcessor = cms.Sequence(postProcessorL1Gen)
+
+dqmRatioTimingPlots = DQMEDHarvester("DQMGenericClient",
+    subDirs = cms.untracked.vstring("L1T/L1TStage2uGT/"),
+    monitorDir = cms.untracked.string(ugtDqmDir),
+    efficiency = cms.vstring(
+       "Ratio_First_Bunch_In_Train 'First Bunch In Train Ratio' first_bunch_in_train den_first_bunch_in_train",
+       "Ratio_Last_Bunch_In_Train 'Last Bunch In Train Ratio' last_bunch_in_train den_last_bunch_in_train",
+       "Ratio_Isolated_Bunch_In_Train 'Isolated Bunch In Train Ratio' isolated_bunch den_isolated_bunch_in_train",
+    ),
+    resolution = cms.vstring(),
+    outputFileName = cms.untracked.string(""),
+    verbose = cms.untracked.uint32(0),
+    inputNum = cms.untracked.string(ugtDqmDir+errHistNumStr),
```

Comment:
The parameters inputNum, inputDen, ratioName, ratioTitle, xAxisTitle, yAxisTitle, and binomialErr are no parameters of DQMGenericClient. Please remove them.
To change the axis labels of the ratio plot you can write for example in the efficiency = cms.vstring():
`"Ratio_First_Bunch_In_Train 'First Bunch In Train Ratio;Bunch crossing number relative to L1A;Algorithm trigger bit' first_bunch_in_train den_first_bunch_in_train"`
Otherwise the axis labels are taken from the input histograms. Better to change them also to fit this scheme.